cmd/init.go

```diff
@@ -94,39 +94,36 @@ func GenerateGraphServer(config *codegen.Config) {
 		os.Exit(1)
 	}
 
-	fmt.Fprintln(os.Stdout, `Exec "go run ./server/server.go" to start GraphQL server`)
+	fmt.Fprintf(os.Stdout, `Exec "go run ./%s" to start GraphQL server`, serverFilename)
 }
 
 func initConfig() *codegen.Config {
 	var config *codegen.Config
 	var err error
 	if configFilename != "" {
 		config, err = codegen.LoadConfig(configFilename)
-		if err != nil {
-			fmt.Fprintln(os.Stderr, err.Error())
-			os.Exit(1)
-		} else if config != nil {
-			fmt.Fprintln(os.Stderr, "config file already exists")
-			os.Exit(0)
-		}
 	} else {
 		config, err = codegen.LoadConfigFromDefaultLocations()
-		if os.IsNotExist(errors.Cause(err)) {
-			if configFilename == "" {
-				configFilename = "gqlgen.yml"
-			}
-			config = codegen.DefaultConfig()
-			config.Resolver = codegen.PackageConfig{
-				Filename: "resolver.go",
-				Type:     "Resolver",
-			}
-		} else if config != nil {
-			fmt.Fprintln(os.Stderr, "config file already exists")
-			os.Exit(0)
-		} else if err != nil {
-			fmt.Fprintln(os.Stderr, err.Error())
-			os.Exit(1)
-		}
+	}
+
+	if config != nil {
+		fmt.Fprintf(os.Stderr, "init failed: a configuration file already exists at %s\n", config.FilePath)
+		os.Exit(1)
+	}
+
+	if !os.IsNotExist(errors.Cause(err)) {
```

Comment:
What if the above code found a config file but there's parsing errors?

Reply:
If there is a parse error then `config` will be `nil` and it will exit with `1` right?  I think that's the correct behaviour.  What do you think should happen?

Reply:
Yeah cool yeah that's what I am thinking too so we will need to handle that edge case then because I don't think 
`!os.IsNotExist(errors.Cause(err))` will catch it.

Reply:
Note the `!` — I know it's a bit hard to read, but it's checking for the opposite case; basically any error _except_ file not found.

Reply:
hrmm okay I guess that makes sense.